Switching jewish cemeteries to SVG

kocio-pl opened the pull request:
Related to https://github.com/gravitystorm/openstreetmap-carto/issues/2045.

Warsaw, z19
Before
![d_qk7zl4](https://user-images.githubusercontent.com/5439713/28763345-a1446a66-75bd-11e7-825d-203cb74114a6.png)
After
![xq1vlchd](https://user-images.githubusercontent.com/5439713/28763353-a6c18578-75bd-11e7-8b4c-6e60924d6c1e.png)

pnorman commented on symbols/grave_yard_jewish.svg:
```diff
@@ -0,0 +1,38 @@
+   version="1.1"
+   width="18"
+   height="18"
+   viewBox="0 0 18 18"
```
needs to be a multiple of 2, for the same reasons as #2716

pnorman commented on symbols/grave_yard_jewish.svg:
```diff
@@ -0,0 +1,38 @@
+     id="canvas"
+     style="visibility:hidden;fill:none;stroke:none" />
+  <path
+     style="fill:#88b78e;fill-opacity:1;stroke-width:0.73192507"
```
It's best to not have stroke

kocio-pl commented:
I guess stroke was not used anyway, but I removed it. Current PNG version is 18 px, but I've changed it to 16 px and now the pattern is a bit more dense:
Before
![o qhp3m6](https://user-images.githubusercontent.com/5439713/28777279-bde517a0-75fa-11e7-83ad-8a645f74937d.png)
After
![mnylwqrf](https://user-images.githubusercontent.com/5439713/28777284-c201bb9a-75fa-11e7-9a8f-eb79762f37c0.png)

pnorman commented:
The icons look bolder now - is this accurate, or just perceptual?

kocio-pl commented:
The color is the same and the size is rather similar, but current images are more blurred than new ones.

kocio-pl commented:
Not needed due to https://github.com/gravitystorm/openstreetmap-carto/pull/2714 work.
